Source/ObjectiveDropboxOfficial/Shared/Handwritten/OAuth/DBOAuth.h

```diff
@@ -140,6 +140,13 @@
 ///
 - (BOOL)clearStoredAccessTokens;
 
+///
+/// When NO, the Dropbox web login page will show a sign up link below the
```

Comment:
@greg-db This is definitely for app store compliance, Apple have requested that we have a sign up flow available and don't force users to first download the Dropbox app. 

I wasn't aware that we were able to navigate to the upgrade page until @scobbe pointed out the situation where that can happen on iPad. I'm going to fix that.

Our reasoning with keeping this as a parameter is to not change things on existing SDK users and to allow us to switch back easily if needed. Once we're sure that our sign up flow is ok on the App Store again I think we can remove this parameter or just change the default.